x-pack/plugins/apm/public/components/alerting/transaction_duration_alert_trigger/index.tsx

```diff
@@ -4,24 +4,31 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { EuiSelect } from '@elastic/eui';
-import { useParams } from 'react-router-dom';
 import { i18n } from '@kbn/i18n';
 import { map } from 'lodash';
 import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useFetcher } from '../../../../../observability/public';
 import { ForLastExpression } from '../../../../../triggers_actions_ui/public';
 import { ALERT_TYPES_CONFIG } from '../../../../common/alert_types';
-import { useEnvironmentsFetcher } from '../../../hooks/use_environments_fetcher';
-import { useUrlParams } from '../../../context/url_params_context/use_url_params';
-import { ServiceAlertTrigger } from '../ServiceAlertTrigger';
-import { PopoverExpression } from '../ServiceAlertTrigger/PopoverExpression';
 import { ENVIRONMENT_ALL } from '../../../../common/environment_filter_values';
+import { TimeSeries } from '../../../../typings/timeseries';
+import { useApmServiceContext } from '../../../context/apm_service/use_apm_service_context';
+import { useUrlParams } from '../../../context/url_params_context/use_url_params';
+import { useEnvironmentsFetcher } from '../../../hooks/use_environments_fetcher';
+import { callApmApi } from '../../../services/rest/createCallApmApi';
+import { getResponseTimeTickFormatter } from '../../shared/charts/transaction_charts/helper';
+import { useFormatter } from '../../shared/charts/transaction_charts/use_formatter';
```

Comment:
This was removed in https://github.com/elastic/kibana/pull/84634/files#diff-6d6da4c7c8ceaea452f6660c70f214671b087f142f869f224514a3213b348427